README.md

```diff
@@ -146,11 +155,23 @@ people = [
           }
       ]
   }
-]
-
+] ###################### needs work
 
+def avg
 coffee_average_per_person = []
+people.each |i| 
+ coffee = 0
```

Comment:
`i` refers to the entire object element of the array 
```
{
      name: 'Ahlam',
      transactions: [
          {
              type: 'COFFEE',
              amount: 7.43
          },
          {
              type: 'TACOS',
              amount: 14.65
          },
          {
              type: 'COFFEE',
              amount: 4.43
          }
      ]
  }
```

so in order to get into the transactions array you should loop through `i[:transactions]` and then check for the type within the transactions array